Jenkinsfile

```diff
@@ -45,5 +45,19 @@ node('docker && awsaccess') {
         reportName: 'cloudformation-ruby-dsl documentation'
       ])
     }
+
+    stage('Push to repository') {
+      def secrets = [
+          [$class: 'VaultSecret', path: 'secret/devops/sonatype_nexus', secretValues: [
+              [$class: 'VaultSecretValue', envVar: 'NEXUS_USER', vaultKey: 'username'],
+              [$class: 'VaultSecretValue', envVar: 'NEXUS_PASSWORD', vaultKey: 'password']]]
+      ]
+
+      if (env.BRANCH_NAME == 'master') {
+        wrap([$class: 'VaultBuildWrapper', vaultSecrets: secrets]) {
+          sh("rake release")
```

Comment:
The nexus repository currently allows overwriting of artifacts. Not 100% sure that's the right behaviour we want right now.